src/instruments/src/Clock/Components/Chrono.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from 'react';
+import { useInteractionEvent } from '../../Common/hooks';
+import { useSimVar } from '../../Common/simVars';
+
+const getDisplayString = (seconds: number | null, running: boolean) : string => (seconds == null ? ''
+    : `${Math.floor(Math.min(seconds, 5999) / 60).toString().padStart(2, '0')}${running ? ':' : ' '}${(Math.floor(Math.min(seconds, 5999) % 60)).toString().padStart(2, '0')}`);
+
+export const Chrono = () => {
+    const [ltsTest] = useSimVar('L:XMLVAR_LTS_Test', 'bool', 250);
+    const [absTime] = useSimVar('E:ABSOLUTE TIME', 'Seconds', 200);
+    const [prevTime, setPrevTime] = useState(absTime);
+
+    const [elapsedTime, setElapsedTime] = useState<null | number>(null);
+    const [running, setRunning] = useState(false);
+
+    useEffect(() => {
+        if (running) {
+            setElapsedTime((elapsedTime || 0) + absTime - prevTime);
```

Comment:
Is this robust to sim time being changed e.g. in the weather pop-up? P3D docs say `ABSOLUTE TIME` is:  
`Time, as referenced from 12:00 AM January 1, 0000`  
I don't know if there is a monotonic clock simvar available.

Reply:
So according to some research by @theomessin, Coherent guarantees well-behaved and monotone behaviour when using the native `setTimeout` and `setInterval`. If `ABSOLUTE TIME` is not the time from outside of the machine we could either do what deltaTime does and use `Date.now()`, or `setTimeout`/`setInterval` respectively.

Reply:
The good thing with using ABSOLUTE TIME is that it respects variable sim rate, which is pretty important IMO, since the real-time isn't really of interest when using the stopwatch/elapsed time.

Reply:
Alright, sounds good. I'm sure QA will test this.